diff --git a/include/osgPPU/Unit.h b/include/osgPPU/Unit.h
index f4fe7c2..0353b1a 100644
--- a/include/osgPPU/Unit.h
+++ b/include/osgPPU/Unit.h
@@ -307,7 +307,8 @@ class OSGPPU_EXPORT Unit : public osg::Group {
         **/
         inline void popFrameBufferObject(osg::State& state)
         {
-            osg::FBOExtensions* ext = osg::FBOExtensions::instance(state.getContextID(), true);
+            //osg::FBOExtensions* ext = osg::FBOExtensions::instance(state.getContextID(), true);
+            osg::GLExtensions* ext = state.get<osg::GLExtensions>();
             ext->glBindFramebuffer(GL_FRAMEBUFFER_EXT, mPushedFBO[state.getContextID()]);
         }
 
